lib/mapit_api.rb

```diff
@@ -1,9 +1,13 @@
 module MapitApi
   class InvalidPostcodeError < StandardError; end
+  class ValidPostcodeNoPlaces < StandardError; end
```

Comment:
I don't think the `MapitApi` module should be raising an error which refers to places, because they're an Imminence concern and no data about `Place`s is returned from MapIt. This error could be called `ValidPostcodeNoLocation` instead, which more closely describes its meaning (following on from @h-lame's comment below about checking for `lat`/`lon` rather than `areas`).